docs/content/troubleshooting/log_status_collect.md

```diff
@@ -1,41 +1,101 @@
+### General status and log collection for IBM Support
+Be sure to run the following steps and copy the output to an external file, when engaging IBM Support and sending log collections.
 
-    ```
-    kubectl get pods -n <namespace> -l csi
-    ```
+1. Check the node status.
+    
+    `kubectl get nodes`
+2. Check the CSI driver component status.
 
-2.  If a pod is not in a _Running_ state, run the following command:
+    `kubectl get -all-namespaces pod -o wide | grep ibm-block-csi`
+3. Check if the PersistentVolumeClaims (PVCs) are _Bound_.
 
-    ```
-    kubectl describe -n <namespace> pod/<pod-name>
-    ```
+    `kubectl get pvc`
 
-    View the logs.
+    **Note:** If the PVCs are not in the _Bound_ state collect the events of all unbound PVCs. (See [Log collection for unbound PVCs](#log-collection-for-unbound-pvcs).)
+
+### Log collection for all CSI driver node pods and their containers
+
+To collect logs for all CSI driver node pods, use the following commands:
+
+    nodepods=`kubectl get pods -l product=ibm-block-csi-driver -l app.kubernetes.io/component=csi-node --output=jsonpath={.items..metadata.name}`
+    
+    for pod in $nodepods;do for container in `kubectl get -n <namespace> pod $pod -o jsonpath='{.spec.containers[*].name}'`;do kubectl logs -n <namespace> $pod -c $container > logs/${pod}_${container}.log;done;done
+
+
+### Log collection for all CSI controller containers
+
+To collect logs for all pods and containers, use the following commands:
+    
+    for container in `kubectl get -n <namespace> pod ibm-block-csi-controller-0 -o jsonpath='{.spec.containers[*].name}'`;do kubectl logs -n <namespace> ibm-block-csi-controller-0 -c $container > logs/ibm-block-csi-controller-0_${container}.log;done
+
+
+### Log collection for CSI operator logs
+To collect CSI operator logs, use the following commands:
+
+    operatorpod=`kubectl get pods --all-namespaces |grep ibm-block-csi-operator|awk '{print $2}'`
+    kubectl logs $operatorpod -n <namespace> > logs/operator
+
+
+### Collecting details of all CSI objects and components
+`kubectl describe all -l product=ibm-block-csi-driver -n <namespace> > logs/describe_ibm-block-csi-driver`
+
+
+### Status collection for CSI pods, daemonset, and statefulset
+`kubectl get all -n <namespace> -l product=ibm-block-csi-driver > logs/get_all_ibm-block-csi-driver`
+
+
+
+### Log collection for the CSI driver controller
+`kubectl logs -f -n <namespace> ibm-block-csi-controller-0 -c ibm-block-csi-controller > logs/ibm-block-csi-controller`
+
+
+### Log collection for the CSI driver node (per worker node or PODID)
+`kubectl logs -f -n <namespace> ibm-block-csi-node-<PODID> -c ibm-block-csi-node > logs/csi-node-<PODID>`
+
+
+
+### Details collection for unbound PVCs
+`kubectl describe pvc <pvc-name> > logs/pvc_not_bounded`
+
+
+
+### Details collection for pods not in the _Running_ state
+`kubectl describe pod <not-running-pod-name> > logs/pod_not_running`
```

Comment:
Also here, consider to add `-n <pod_namespace>`

Reply:
Please write the whole command with the -n <pvcs_namespace> so I see exactly where to place it in the command. thanks
@roysahar

Reply:
done